build.gradle

```diff
@@ -83,7 +83,16 @@ allprojects {
     }
 
     project.getExtensions().getExtraProperties().set("versions", VersionProperties.getVersions());
+}
+
+configurations {
```

Comment:
Just curious, if we are already doing this in the root gradle file, do we still need to do the same for all the modules?